src/components/forms/NetworkDevicesForm/Acls.tsx

```diff
@@ -0,0 +1,111 @@
+import type { FC } from "react";
+import type { LxdNicDevice } from "types/device";
+import type { LxdNetwork } from "types/network";
+import type { CustomNetworkDevice } from "util/formDevices";
+import type { InstanceAndProfileFormikProps } from "components/forms/instanceAndProfileFormValues";
+import ExpandableList from "components/ExpandableList";
+import ResourceLink from "components/ResourceLink";
+import NetworkAclSelector from "pages/networks/forms/NetworkAclSelector";
+
+interface Props {
+  project: string;
+  managedNetworks: LxdNetwork[];
+  device: LxdNicDevice | CustomNetworkDevice;
+  readOnly?: boolean;
+  formik?: InstanceAndProfileFormikProps;
+  index?: number;
+  canSelectManualAcls?: boolean;
+}
+
+const Acls: FC<Props> = ({
+  project,
+  managedNetworks,
+  device,
+  readOnly,
+  formik,
+  index,
+  canSelectManualAcls,
+}) => {
+  const getNetworkAcls = (networkName: string) => {
+    const network = managedNetworks.find((t) => t.name === networkName);
+    if (network) {
+      return network.config["security.acls"]?.split(",") || [];
+    }
+    return [];
+  };
+
+  const getDeviceAcls = (device: LxdNicDevice | CustomNetworkDevice) => {
+    return (device as LxdNicDevice)["security.acls"]?.split(",") || [];
+  };
+
+  const networkAcls = getNetworkAcls((device as LxdNicDevice).network);
+  const deviceAcls = getDeviceAcls(device);
+
+  return (
+    <>
+      {networkAcls.length > 0 && (
+        <div className="acls-from-network">
+          ACLs from network
+          <div className="acls-list">
+            <ExpandableList
+              items={networkAcls.map((acl) => (
+                <div key={acl} className="u-whitespace-nowrap">
+                  <ResourceLink
+                    type="network-acl"
+                    value={acl}
+                    to={`/ui/project/${encodeURIComponent(project || "default")}/network-acl/${encodeURIComponent(acl)}`}
+                  />
+                </div>
+              ))}
+            />
+          </div>
+        </div>
+      )}
+      {(deviceAcls.length > 0 ||
+        (!readOnly &&
+          formik &&
+          index !== undefined &&
+          canSelectManualAcls)) && (
+        <div>
+          Manual ACLs
```

Comment:
In edit mode, this should be a label for the input below.

Reply:
> MultiSelect doesn't have label as argument

Good point, this is an issue also in the network config page, where the label for ACLs is missing the connection. Yes, I think having a way to set the `id` of the `MultiSelect` is a good improvement and would unblock the fix here and in the network config.

> And then what should "Manual ACLs" be in read only mode ? A span ?

Yes, a span in read mode. For read mode I was thinking we just present one heading and listing all ACLs, so not communicating the split between "from network" and "from nic device" as a source -- to simplify read mode.